test/projects/test_lps.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from runner import ProjectType
+from glotter import project_test, project_fixture
+from test.utilities import clean_list
+
+invalid_permutations = (
+    'description,in_params,expected', [
+        (
+            'no input',
+            None,
+            'Incorrect input provided. Program Terminated'
+        ), (
+            'empty input',
+            "",
+            'Incorrect input provided. Program Terminated'
+        ), (
+            'invalid input: no palindromic present',
+            "Polip",
+            'No Palindromic substring present.'
+        ),
+        (
+            'invalid input: no palindromic present',
```

Comment:
Did you mean to duplicate this test?

Reply:
No I didn't, resolved in https://github.com/TheRenegadeCoder/sample-programs/pull/2275/commits/7f6bc9997750756882e89a3bbb295633a3bbd5ea.